WordPress/Classes/Networking/ServiceRemoteREST.m

```diff
@@ -0,0 +1,20 @@
+#import "ServiceRemoteREST.h"
+#import "WordPressComApi.h"
+
+@interface ServiceRemoteREST ()
+@end
+
+@implementation ServiceRemoteREST
+
+- (id)initWithApi:(WordPressComApi *)api {
+    
+    NSAssert([api isKindOfClass:[WordPressComApi class]],
```

Comment:
To be honest they're all sort of the same, with the added bonus that `NSParameterAssert` has its own standardized message.  In fact if you check its definition, it's not much more than that.

That said, I think that even though the difference is slight and subtle, it's a good thing to have standardized messages for parameter assertion failures.